packages/create-sitecore-jss/src/templates/nextjs-styleguide/package.json

```diff
@@ -7,6 +7,7 @@
     "@sitecore-jss/sitecore-jss-dev-tools": "^20.0.0-canary"
   },
   "scripts": {
+    "bootstrap": "ts-node --project tsconfig.scripts.json scripts/bootstrap.ts && graphql-let",
```

Comment:
We should be performing the `&& graphql-let` on the base template (then don't need this bootstrap script here). Even though we don't have any queries to code-gen, I think the expectation should be that it's ready to do so when you start adding code.

Reply:
Right, I didn't know that we decided to have this command if we don't have gql queries. As I remember it will fail if we don't have gql queries